lib/benchee/formatters/console.ex

```diff
@@ -145,11 +155,65 @@ defmodule Benchee.Formatters.Console do
     units = Conversion.units(sorted_scenarios, scaling_strategy)
     label_width = label_width(sorted_scenarios)
 
-    [column_descriptors(label_width) |
+    output = [column_descriptors(label_width) |
       scenario_reports(sorted_scenarios, units, label_width)
       ++ comparison_report(sorted_scenarios, units, label_width, config)]
+
+    if config.extended_options do
+      output ++ [descriptor("Extended options") |
```

Comment:
here and in method names I think I prefer calling it extended statistics, not options :)

Reply:
Agreed, "Extended options" isn't as explanatory as "Extended statistics".